pkg/minikube/out/out.go

```diff
@@ -113,28 +115,38 @@ func Styled(st style.Enum, format string, a ...V) {
 	}
 }
 
-func boxedCommon(printFunc func(format string, a ...interface{}), format string, a ...V) {
+func boxedCommon(printFunc func(format string, a ...interface{}), cfg box.Config, title string, format string, a ...V) {
 	str := Sprintf(style.None, format, a...)
 	str = strings.TrimSpace(str)
-	box := box.New(box.Config{Py: 1, Px: 4, Type: "Round"})
-	if useColor {
-		box.Config.Color = "Red"
+	box := box.New(cfg)
+	if !useColor {
+		box.Config.Color = ""
 	}
-	str = box.String("", str)
+	str = box.String(title, str)
 	lines := strings.Split(str, "\n")
 	for _, line := range lines {
 		printFunc(line + "\n")
 	}
 }
 
-// Boxed writes a stylized and templated message in a box to stdout
+// Boxed writes a stylized and templated message in a box to stdout using the default style config
 func Boxed(format string, a ...V) {
-	boxedCommon(String, format, a...)
+	boxedCommon(String, defaultBoxCfg, "", format, a...)
 }
 
-// BoxedErr writes a stylized and templated message in a box to stderr
+// BoxedErr writes a stylized and templated message in a box to stderr using the default style config
 func BoxedErr(format string, a ...V) {
-	boxedCommon(Err, format, a...)
+	boxedCommon(Err, defaultBoxCfg, "", format, a...)
+}
+
+// BoxedWithConfig writes a templated message in a box with customized style config to stdout
+func BoxedWithConfig(cfg box.Config, st style.Enum, title string, format string, a ...V) {
```

Comment:
this is currently not being used anywhere in the code, and the rule of thumb is not to keep code never used anywhere...how about find a good place/example for this func to be used ?

Reply:
Yes this part of the `minikube tips` feature. Please see my proposal: https://github.com/kubernetes/minikube/pull/11696

I was trying to create smaller PRs for easier review. But I can have all the changes in one PR as well if it makes more sense.